### DISCUSSION TITLE
Disabling the automatic cache update?

### DISCUSSION THREAD

**Discussion by CaptainFrosty** [tldr-pages/tldr-node-client#375] [Q&A] [answered]
Hi, is it possible to turn off the cache update that happens every time a page isn't found? It takes a few minutes to finish on my server for some reason, (I don't know why) and ~90% of the time I'm triggering it with a typo, not an unknown command. 

I'm using nodejs version 3.3.7.

**Comment by kbdharun** [comment, marked as answer]
Sorry for the late reply, you can indeed disable auto-updating when a page is not found by updating the [configuration](https://github.com/tldr-pages/tldr-node-client#configuration) file and setting this variable `skipUpdateWhenPageNotFound` to true.
